src/Symfony/Component/DependencyInjection/Compiler/AutowirePass.php

```diff
@@ -159,6 +159,10 @@ private function getMethodsToAutowire(\ReflectionClass $reflectionClass, array $
                     continue 2;
                 }
             }
+
+            if ($reflectionMethod->isAbstract() && !$reflectionMethod->getNumberOfParameters()) {
```

Comment:
that won't provide anything more to me since those checks are going to happen anyway later on